wasm-runtime/polkadot/src/runtime/staking.rs

```diff
@@ -119,6 +82,25 @@ pub fn unstake(transactor: &AccountID) {
 	(current_era() + bonding_duration()).store(&transactor.to_keyed_vec(b"sta:bon:"));
 }
 
+// PRIVILEDGED API
+
+/// Set the number of sessions in an era.
+pub fn set_sessions_per_era(new: BlockNumber) {
```

Comment:
`priviledged::set_sessions_per_era()` would imho look nicer and would made it more visible to see that it's a priviledged API (especially at call site) than the comment above.

Reply:
done in the latest PR.